src/main/webapp/app/exam/participate/exam-cover/exam-participation-cover.component.html

```diff
@@ -15,86 +15,86 @@ <h2 class="col-md-6" style="text-align: center; font-weight: normal" id="exam-fi
-                <div class="row justify-content-center" style="padding-bottom: 0; margin-bottom: 0">
-                    <div class="form-group" style="padding-bottom: 0; margin-bottom: 0; width: unset">
+            </div>
+            <hr />
+            <jhi-exam-start-information [exam]="exam" [studentExam]="studentExam" />
+            <span [innerHTML]="formattedGeneralInformation"></span>
+            <div class="d-inline-flex align-items-center mb-5">
+                <div class="ps-1">
+                    <input
+                        [(ngModel)]="confirmed"
+                        type="checkbox"
+                        id="confirmBox"
+                        (click)="updateConfirmation()"
+                        class="form-check-input me-2"
+                        [class.ms-0]="!this.exam.confirmationStartText"
+                        [required]="inserted"
+                        [disabled]="waitingForExamStart"
+                    />
+                    <label for="confirmBox" id="formatted-confirmation-text" class="form-check-label" [innerHTML]="formattedConfirmationText"></label>
+                </div>
+                @if (!formattedConfirmationText) {
+                    <div class="login-form">
+                        <label class="font-weight-bold" for="confirmBox" jhiTranslate="{{ 'artemisApp.exam.startCheckBoxText' }}"></label>
+                    </div>
+                }
+            </div>
+
+            <div class="mb-4 fw-bold" jhiTranslate="artemisApp.exam.startConsentText"></div>
+            <div class="login-form mb-4">
+                <div class="row pb-0 mb-0">
+                    <div class="form-group pb-0 mb-0" style="width: unset">
                         <input
-                            size="50"
+                            size="40"
                             type="text"
                             class="form-control"
                             name="fullname"
                             id="fullname"
                             [placeholder]="'artemisApp.examParticipation.namePlaceholder' | artemisTranslate"
                             [(ngModel)]="enteredName"
                             [ngModelOptions]="{ updateOn: 'change' }"
-                            [disabled]="startView ? waitingForExamStart : false"
+                            [disabled]="waitingForExamStart"
                         />
                     </div>
                 </div>
-                <div class="row justify-content-center">
-                    <div id="your-name" class="col-md-6" style="font-weight: lighter">
-                        <span> ({{ 'artemisApp.examParticipation.nameDisplay' | artemisTranslate: { fullName: accountName } }}) </span>
+                <div class="row">
+                    <div id="your-name" class="md-6 fw-lighter">
+                        <span> {{ 'artemisApp.examParticipation.nameDisplay' | artemisTranslate: { fullName: accountName } }} </span>
```

Comment:
@pzdr7 Suggested to use a span with jhiTranslate. In order to make it consistent, could we use it here as well?

Reply:
Sure, i changed it to jhiTranslate now. But I think there is now a PR that changes all old translation pipes to jhiTranslate here #8803.